tests/artifacts/predefined_data_configs/multiple_datasets_with_odm.yaml

```diff
@@ -0,0 +1,70 @@
+dataprocessor:
+    type: odm
+    sampling_stopping_strategy: first_exhausted # ignored
+    seed: 66
+    odm:
+      update_interval: 1 # update every step
+      sampling_interval: 1 # sample category for every sample
+      reward_type: validation_loss # uses eval loss of each dataset as reward
+      gamma: 0.1 # MAB hyper-parameter
+      eta: 0.2 # MAB hyper-parameter
+datasets:
+  - name: dataset_1
+    split:
+      train: 0.8
+      validation: 0.2
```

Comment:
Can you add a comment after the ratio that this will be overloaded as a reward dataset too in odm.